teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
```

Comment:
- [ ] You will need to import the Person class here for this inheritance to work.